jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedThreadMXBeanImpl.java

```diff
@@ -100,6 +100,29 @@ public long[] getThreadAllocatedBytes(long[] threadIds) {
 		return allocatedBytes;
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public long getTotalThreadAllocatedBytes() {
+		if (isThreadAllocatedMemorySupported()) {
+			if (isThreadAllocatedMemoryEnabled()) {
```

Comment:
I suggest that inverting these two `if` statements improves readability.

Reply:
Updated for both review comments.